src/components/Error/ErrorView.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter as Router, useHistory } from 'react-router-dom';
+import {
+  Button,
+  Loader,
+  Logo,
+  presetGpnDark,
+  Text,
+  Theme,
+  useInterval,
+  useMount,
+  useUnmount,
+} from '@gpn-prototypes/vega-ui';
+import cn from 'bem-cn';
+
+import { ServerError } from '../../utils/graphql-client';
+
+import { GazpromLogo } from './GazpromLogo';
+
+import './Error.css';
+
+type ErrorViewProps = ServerError;
+
+const TIME_TO_RELOAD = 60;
```

Comment:
```suggestion
const TIME_TO_RELOAD_SEC = 60;
```

лучше всегда указывать в каких единицах измерения хранится значение